aerialist/px4/local_agent.py

```diff
@@ -25,7 +26,14 @@ def run(self):
                 self.drone.run_scheduled()
             log = self.simulator.get_log()
             if self.config.agent is not None and self.config.agent.path is not None:
-                file_helper.upload(log, self.config.agent.path)
+                upload_string = config("UPLOAD_LIST", default="")
```

Comment:
let's keep uploading the unzipped log to the cloud. 
let's stract a helper function for the whole thing and put in the filehelper.